server/controller.js

```diff
@@ -1,5 +1,81 @@
 const model = require('./model.js');
+const client = require('./db.js');
+
 
 module.exports = {
+  addUser: async (req, res) => {
+    const user = req.body; // expect body to contain full user info
+    if (user.userType === 'seeker') {
+      try {
+        await model.addSeeker(user);
+        res.sendStatus(201);
+      } catch(e) {
+        console.log('eeeeee', e);
+      }
+    } else if (user.userType === 'recruiter') {
+      try {
+        await model.addRecruiter(user);
+        res.sendStatus(201);
+      } catch(e) {
+        console.log('eeeeee', e);
+      }
```

Comment:
Do you need to add a link to a model here that posts to the Firebase table also? Maybe just in the addSeeker/addRecruiter model?

Reply:
I've added 'addToFirebase' inside model. So now when creating a user, it will based on userType, either call (modal.addSeeker + modal.addToFirebase) or (modal.addRecruiter + modal.addToFirebase). That should handle it.